bsp_diff/caas/device/intel/civ/host/vm-manager/0025-Add-some-scripts.patch

```diff
@@ -0,0 +1,183 @@
+From ea792cb12ed2c27760e64b875119cf7ecc305ba0 Mon Sep 17 00:00:00 2001
+From: "Qi, Yadong" <yadong.qi@intel.com>
+Date: Mon, 27 Jul 2020 13:23:46 +0800
+Subject: [PATCH 3/3] Add some scripts
+
+Add some scripts
+
+Tracked-On: OAM-91886
```

Comment:
1) OAM-91886 is a very old ticket.
It doesn't match the changes here. Please check

2) Why do we need this patch? Why is it not in horizontal?